### ISSUE TITLE
NTT and add-on compatibility for Nightly 56

### ISSUE THREAD

**Issue by harl** [mozilla/nightlytt#241]
NTT's option to force add-on compatibility doesn't seem to have the desired effect for me in Nightly 56.
I'm trying to transfer the add-ons most of which I'm actively using, from a rather oldish FF/Nightly profile to a new one to get rid of one or two UI issues/changes that got stuck sometime during the last couple months.

e10s is enabled and working in both profiles.
Some add-ons seem to install fine in the new profile even though they are marked as "LEGACY" add-ons in the old profile, like Tab Mix Plus, whether the various checkCompatibility and disableCheckCompatibility variables are false or true.
Some add-ons are simply declined by the new profile because it deems them "incompatible", no matter what.  I've enabled NTT's option to force add-on compatibility and it doesn't change anything for those add-ons.
I've set xpinstall.signatures.required to false.
I've tried editing install.rdfs only to see a message stating the add-on was corrupt (and I'm not planning on editing the install.rdfs of about 50 add-ons, so this was purely for testing).

The same add-ons work fine with my old profile, so I KNOW the same version of Nightly can be forced to load those same add-on versions, yet I haven't managed to get the new profile to accept them.

Bottom line is:  I expected NTT's compatibility option to make this possible, but it seems different.
I've used NTT before.  And something is missing this time.

**Comment by whimboo** [comment]
With Firefox 57 all legacy addons including XUL-based and AddonSDk-based extensions are no longer supported. As such this feature will not be necessary anymore. As such we won't support that feature anymore with the rewrite of the NTT extension, which is available as beta on AMO now.
